app/notification/application/map_contents.py

```diff
@@ -33,11 +35,11 @@ def from_json(json_data: dict):
         Returns:
             ApplicationData object with application contents.
         """
-        data = Notification.from_json(json_data)
+        current_app.logger.info("Mapping contents for APPLICATION")
         application = data.content["application"]
         return Application(
             contact_info=data.contact_info,
-            questions=Application.process_questions_and_answers(data),
+            questions=Application.bytes_object_for_questions_answers(data),
             submission_date=application.get("date_submitted"),
```

Comment:
Were you going to use the constants here for field names?

Reply:
Yes, for that I need to speak with you around utils service so I haven't updated it yet. Huddle?